### PR TITLE
use doctr to also manage release docs

### PR DESCRIPTION
**Pull request by gforsyth**
now that we have a release that contains the encrypted deploy key, we
can also have doctr always push the most recent release docs to
`xonsh-docs`.

there may be more fiddling to do to get a documentation selector
working but this is a start

Usual caveat that we won't know if this is working until it's merged, but if anyone spots any obvious problems in the yaml logic, let me know.

### CODE REVIEW

**Comment by astronouth7303** [comment]
Go for it and see if it breaks something?

**Comment by gforsyth** [comment]
No time like the present

File: .travis.yml [hunk 1]
```diff
@@ -53,13 +53,12 @@ script:
       make html;
       cd ..;
       doctr deploy --deploy-repo xonsh/xonsh-docs --gh-pages-docs dev;
+      git checkout $(git describe --tags `git rev-list --tags --max-count=1`);
+      cd docs;
+      make html;
```
Comment (asmeurer) [inline .travis.yml]: You might want to put a `make clean` here to clear out any cruft from the dev build above.

**Comment by scopatz** [comment]
Thanks for this @gforsyth! Just the one small change @asmeurer suggested.

**Comment by gforsyth** [comment]
Good call, @asmeurer. Thanks.  Updated.

**Comment by scopatz** [comment]
Thanks @gforsyth! Also, I am not sure if you avoided this on purpose, but you can also do `make clean html`

**Pull request merged**